### ISSUE TITLE
Tortoise on Xaero's MiniMap are displaying as invalid icons

### ISSUE THREAD

**Issue by RashidFTW** [BrenBronze/brenbronze-MC-Project-Issues-N-Suggestions#1]
**Project**
Xaero's Naturalist Icons

**Project version**
Xaeros Naturalist Icons - 1.20.1.zip

**Project release**
?? Xaeros Naturalist Icons - 1.20.1.zip

**Describe your issue**
Tortoise on Xaero's MiniMap are displaying as invalid icons (black and purple, see image attached)

**To reproduce issue..**
Just install the resource pack, and spawn in a tortoise

**Media**
![image](https://github.com/BrenBronze/brenbronze-MC-Project-Issues-N-Suggestions/assets/160150516/7aee86f8-dda0-4157-92b0-4aff5e49aa87)

**Additional context**
Add any other context about the issue here.
Umm, playing modded Fabric 1.20.1

**Comment by RashidFTW** [comment]
I decided to inspect the resource pack to see what's wrong. I found that the file names and the string in the definitions .json don't match.

![image](https://github.com/BrenBronze/brenbronze-MC-Project-Issues-N-Suggestions/assets/160150516/be89b6b9-5e98-4390-9f14-3afbe4aacec5)
![image](https://github.com/BrenBronze/brenbronze-MC-Project-Issues-N-Suggestions/assets/160150516/a283543c-6cd3-44c7-9c41-ae2e5a82a28f)

After renaming the files, the issue got resolved, seems it was a simple typo on your part.
